scripts/apps/authoring/styles/multiedit.scss

```diff
@@ -20,9 +20,13 @@ $top-offset: 48px;
 }
 
 .multiedit-screen {
+    height: 90%;
```

Comment:
why 90% and not 100%? Is this to account for bottom toolbar? If so, it'd be better to use `calc` function to subtract the bottom toolbar height.

Reply:
I have made some changes so that it looks good, I have align the burger menu and remove and save button on the same line.